### PR TITLE
ADC1&2 working with DMA and timer 2 and 3

### PR DESCRIPTION
**Pull request by mariuscrsn**

### CODE REVIEW

File: firmware/Sources/HAL/HAL_ADC/hal_adc.c [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+	else
+		res = (_meas_pwr[(i - 1) / 2] + _meas_pwr[i / 2]) / 2;
+  return res;
+}
+
+
+/**********************************************************************************/
+/*                        Definition of exported functions                        */
+/**********************************************************************************/
+void HAL_Adc1_Callback(void){
```
Comment (pastorpflores) [inline firmware/Sources/HAL/HAL_ADC/hal_adc.c:120]: is possible to follow naming convention?

File: firmware/Sources/HAL/HAL_ADC/hal_adc.c [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+	_idx_cplt_ADC1 = _ADC1_BUFFER_SIZE/2 - _idx_cplt_ADC1;
+}
+
+void HAL_Adc2_Callback(void){
```
Comment (pastorpflores) [inline firmware/Sources/HAL/HAL_ADC/hal_adc.c:124]: is possible to follow naming convention?

File: firmware/Sources/HAL/HAL_ADC/hal_adc.h [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+ * @enum HAL_ADC_port_e
+ * @brief ADC channels.
+ */
+typedef enum
+{
+	HAL_ADC_LS_CURR,			/** ADC2_IN1 - PA4 **/
+	HAL_ADC_LS_VOLT,			/** ADC2_IN2 - PA5 **/
+	HAL_ADC_HS_VOLT,			/** ADC2_IN4 - PA6 **/
+
+	HAL_ADC_EXT_TEMP1,		/** ADC1_IN1 - PA5 **/
```
Comment (pastorpflores) [inline firmware/Sources/HAL/HAL_ADC/hal_adc.h:52]: 3 temps?
Should you asign a more redeable name?

**Review by pastorpflores — CHANGES_REQUESTED** [review]

File: firmware/Sources/HAL/HAL_ADC/hal_adc.c (hunk 2 above)
Comment (mariuscrsn) [reply firmware/Sources/HAL/HAL_ADC/hal_adc.c:124]: Done

File: firmware/Sources/HAL/HAL_ADC/hal_adc.h (hunk 3 above)
Comment (mariuscrsn) [reply firmware/Sources/HAL/HAL_ADC/hal_adc.h]: Done

**Review by pastorpflores — APPROVED** [review]

**Pull request merged**